### PR TITLE
Add hostprocess containerd logger

### PR DESCRIPTION
**Pull request by jsturtevant**
**Reason for PR**:

Containerd by default logs to ETW when installed as a service. This provides a way to dynamically capture those logs via a config map configuration.  By using https://github.com/Azure/diagnostics-eventflow can also configure other [outputs ](https://github.com/Azure/diagnostics-eventflow#outputs)for these files.

**Issue Fixed**:

Issue #

**Requirements**

- [ ] Sqaush commits 
- [ ] Documentation
- [ ] Tests

**Notes**:

### CODE REVIEW

File: hostprocess/containerd-logger/readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Containerd logger
+
+This is a small applicaiton that you can run inside a hostproces container (or directly on the host) that will log ETW events to stdout.
```
Comment (knabben) [inline hostprocess/containerd-logger/readme.md:3]:
```suggestion
This is a small application that you can run inside a hostproces container (or directly on the host) that will log ETW events to stdout.
```

File: hostprocess/containerd-logger/readme.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+This makes use of the [EventFlow library](https://github.com/Azure/diagnostics-eventflow).  The [inputs](https://github.com/Azure/diagnostics-eventflow#inputs) and [outputs](https://github.com/Azure/diagnostics-eventflow#outputs) can be adjust according to the documentation. There are also [filters](https://github.com/Azure/diagnostics-eventflow#filters) that can be applied to events.
+
+## Hostpocess container
+
+Build the contianer:
```
Comment (knabben) [inline hostprocess/containerd-logger/readme.md:9]:
```suggestion
Build the container:
```

File: hostprocess/containerd-logger/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# syntax=docker/dockerfile:1
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy everything else and build
+COPY program.cs ./
```
Comment (knabben) [inline hostprocess/containerd-logger/Dockerfile:10]:
```suggestion
COPY Program.cs ./
```

File: hostprocess/containerd-logger/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+COPY eventFlowConfig.json ./
+RUN dotnet publish -c Release -r win-x64 -o out
+
+# Build runtime image
+FROM mcr.microsoft.com/windows/nanoserver:1809
+WORKDIR /
+COPY --from=build-env /app/out/containerd-logger.exe .
```
Comment (knabben) [inline hostprocess/containerd-logger/Dockerfile:17]:
```suggestion
COPY eventFlowConfig.json ./
```

Comment (knabben) [inline hostprocess/containerd-logger/Dockerfile:17]:
```suggestion
COPY --from=build-env /app/out/containerd-logger.exe .
COPY --from=build-env eventFlowConfig.json ./
```

File: hostprocess/containerd-logger/readme.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+```
```
Comment (knabben) [inline hostprocess/containerd-logger/readme.md:11]:
```suggestion
cd containerd-logger
```

Comment (knabben) [inline hostprocess/containerd-logger/readme.md:11]:
```suggestion
```
cd containerd-logger
```

File: hostprocess/containerd-logger/readme.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+docker build -t jsturtevant/containerd-logger .
```
Comment (knabben) [inline hostprocess/containerd-logger/readme.md:12]:
```suggestion
cd containerd-logger
docker build -t jsturtevant/containerd-logger .
```

File: hostprocess/containerd-logger/Dockerfile (hunk 4 above)
Comment (jsturtevant) [reply hostprocess/containerd-logger/Dockerfile:17]: I left coping the `eventflowConfig.json` out here since it is copied over via config map.

**Pull request closed without merging**